test/integration/addons_test.go

```diff
@@ -68,6 +69,46 @@ func testDashboard(t *testing.T) {
 	}
 }
 
+func testIngressController(t *testing.T) {
+	t.Parallel()
+	minikubeRunner := NewMinikubeRunner(t)
+	kubectlRunner := util.NewKubectlRunner(t)
+
+	minikubeRunner.RunCommand("addons enable ingress", true)
```

Comment:
for future work, we should utilize the `minikube cache` command to make sure that the ingress controller image is cached for our tests.

Reply:
@r2d4 Okay, I'll make this feature.

Reply:
Added this feature from #2272.